### PR TITLE
{bio}[iccifort/2019.1.144-GCC-8.2.0-2.31.1] bwa-meth v0.2.2, BWA v0.7.17 w/ Python 2.7.15 + 3.7.2

### PR DESCRIPTION
**Pull request by Darkless012**
(created using `eb --new-pr`)

### CODE REVIEW

**Comment by Darkless012** [comment]
Unfortunately they don't have https enabled on their website.
http://bio-bwa.sourceforge.net/
https://bio-bwa.sourceforge.net/

**Comment by boegel** [comment]
@Darkless012 It's actually tripping over the `http://` URL in the header (but it shouldn't be, see #9180)...

Should no longer be a holdup once #9224 gets merged.

File: easybuild/easyconfigs/b/bwa-meth/bwa-meth-0.2.2-iccifort-2019.1.144-GCC-8.2.0-2.31.1.eb [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+        'checksums': ['9541d45f14f9c9ee665289fddc2a40135a8b7abdb600acd85d6c074475ce8238'],
+    }),
+    (name, version, {
+        'source_tmpl': 'v%s.tar.gz' % version,
+        'source_urls': ['https://github.com/brentp/bwa-meth/archive'],
+        'checksums': ['b7284f016be0a99486219b272d90bda47ee28969ea7ef9f28c701b23f2a64955'],
+        'modulename': 'bwameth',
+    }),
+]
+
```
Comment (boegel) [inline easybuild/easyconfigs/b/bwa-meth/bwa-meth-0.2.2-iccifort-2019.1.144-GCC-8.2.0-2.31.1.eb]: @Darkless012 You'll need to fix the shebang in the Python script:

```
$ head -1 /software/bwa-meth/0.2.2-iccifort-2019.1.144-GCC-8.2.0-2.31.1/bin/{bwameth.py,toolshed}
==> /software/bwa-meth/0.2.2-iccifort-2019.1.144-GCC-8.2.0-2.31.1/bin/bwameth.py <==
#!/software/Python/2.7.15-GCCcore-8.2.0/bin/python

==> /software/bwa-meth/0.2.2-iccifort-2019.1.144-GCC-8.2.0-2.31.1/bin/toolshed <==
#!/software/Python/2.7.15-GCCcore-8.2.0/bin/python
```

Can we also add sanity check commands for both these tools with `--help` (which would have caught this issue)?

**Comment by boegel** [comment]
(close/re-open to verify fix merged in #9224)

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 2 out of 2 (2 easyconfigs in this PR)
node2598.golett.os - Linux centos linux 7.7.1908, Intel(R) Xeon(R) CPU E5-2680 v3 @ 2.50GHz, Python 2.7.5
See https://gist.github.com/d35000c262c48f626c5c289d406e0e3c for a full test report.

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 2 out of 2 (2 easyconfigs in this PR)
node3204.victini.os - Linux centos linux 7.6.1810, Intel(R) Xeon(R) Gold 6140 CPU @ 2.30GHz, Python 2.7.5
See https://gist.github.com/15b9adea36fba281d6eeaa8415af6893 for a full test report.

**Comment by boegel** [comment]
Going in, thanks @Darkless012!

**Pull request merged**